src/index.ts

```diff
@@ -57,3 +57,11 @@ export function loadInstalledBeautifiers(): Promise<Unibeautify> {
 export function setupUnibeautify(): Promise<Unibeautify> {
   return loadInstalledBeautifiers();
 }
+
+// TODO filter languages based on beautifiers that are installed
+export function getSupportedLanguages(): string[] {
```

Comment:
Isn't there a Unibeautify core method for this?

Reply:
Yea, https://github.com/Unibeautify/unibeautify/blob/master/src/beautifier.ts#L279-L283.  Was wanting to see how it looks outputting a lot of lines.